### PR TITLE
MEX-112: Make Lab Tests into a free text box

### PR DESCRIPTION
**Pull request by brandones**
https://tickets.pih-emr.org/browse/MEX-112

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/visit/constants.js [hunk 1]
```diff
@@ -276,12 +276,15 @@ angular.module('constants', [])
         normalExamFinding: {
             uuid: "3cd750a0-26fe-102b-80cb-0017a47871b2"
         },
-        labTestOrdered: {
+        medicationOrders: {  // 1282
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/scripts/visit/constants.js]: Why the change here @brandones? Was this a typo on our part you found?

Comment (brandones) [reply omod/src/main/webapp/resources/scripts/visit/constants.js]: I think it had just been named the wrong thing by accident: https://concepts.pih-emr.org/openmrs/dictionary/concept.htm?conceptId=1282

Comment (lnball) [reply omod/src/main/webapp/resources/scripts/visit/constants.js]: Thanks.  Sounds like a good find.
